packages/rum-core/test/common/compress.spec.js

```diff
@@ -270,4 +271,45 @@ describe('Compress', function() {
 
     testMappedObject(error, compressed)
   })
+
+  it('should compress payload', async () => {
```

Comment:
Maybe add a test and patch `CompressionStream` with a function that throws an error to cover catch part.

Reply:
It was really hard to do this part, Blob consumes any data its given and compression stream wont throw, i tried multiple things but failed. Thats why didn't add a test, didn't want to do stub as it doesn't make sense.